src/recipe/ingredients.ts

```diff
@@ -49,8 +56,48 @@ function parse_ingredients(content: string): Ingredient[] {
     return recipes;
 }
 
+function parse_ingredients_recipemd(content: string): Ingredient[] {
+    const debug = false;
+    const recipes: Ingredient[] = new Array();
+    var ingredients;
+    
+
+    if (content.substring(0,2)=='---') {
+      ingredients = content.split('---')[3];
+    } else {
+      ingredients = content.split('---')[1];
+    }
+
+    if (debug) {
+      console.log("INGREDIENTS", ingredients);
+    }
+
+
+    if (typeof ingredients == 'undefined' || ingredients.length<=0) {
+        return new Array();
+    }
+    if (debug) {
+      console.log("PROCESSING", ingredients);
```

Comment:
Same here



If the log statements are going to be kept, could you add the function name to the beginning of the message